JudoPayDotNet/Models/PaymentModel.cs

```diff
@@ -49,6 +49,15 @@ protected PaymentModel()
         [DataMember(EmitDefaultValue = false)]
         public IDictionary<string, string> YourPaymentMetaData { get; set; }
 
+        /// <summary>
+        /// Gets or sets the payment meta data.
+        /// </summary>
+        /// <value>
+        /// Payment meta data.
+        /// </value>
+        [DataMember(EmitDefaultValue = false)]
+        public IDictionary<string, string> PaymentMetaData { get; set; }
```

Comment:
What's the difference between `YourPaymentMetaData` and `PaymentMetaData`?

Reply:
**PaymentMetaData** is a new JSON payload automatically generated by our mobile SDKs to help support additional data requirements for 3D Secure 2.0.  **YourPaymentMetaData** is used by merchants to add their own data about the transaction to help with their own reconciliation.  **YourPaymentMetaData** will likely be renamed in future API changes,